api/openapi_server/openapi/paths/guestApplicationManagement.yaml

```diff
@@ -0,0 +1,22 @@
+get:
+  summary: Get a list of tasks to complete application.
+  tags:
+    - guest_dashboard
+  operationId: get_guest_dashboard_by_id_and_application_id
```

Comment:
i would rename this. 'dashboard' can imply a lot of things but the summary is very specific about getting tasks for an application. maybe, 
get_guest_application_tasks